src/main/xar-resources/data/upgrading.xml

```diff
@@ -17,207 +17,85 @@
-            <para>The 2.2 release is not binary compatible with the 1.4.x series. You need to
-                backup/restore. If you experience problems with user logins after the restore, please restart eXist-db.</para>
-            <para>2.2 introduces a <emphasis>new range index module</emphasis>. Old index definitions will still work though as we made sure to keep backwards
-                compatible. If you would like to upgrade to the new index, check its <ulink url="newrangeindex.xml">documentation</ulink>.</para>
-            <para>The XQuery engine has been updated to support the changed syntax for <emphasis>maps in XQuery 3.1</emphasis>. The query parser will still accept the
-                old syntax for map constructors though (<code>map { x:= "y"}</code> instead of <code>map { x: "y" }</code> in XQuery 3.1), so old
-                code should run without modifications. All map module functions from XQuery 3.1 are 
-                <ulink url="{fundocs}/view.html?uri=http://www.w3.org/2005/xpath-functions/map&amp;location=java:org.exist.xquery.functions.map.MapModule">available</ulink>.</para>
-            <para>The signatures for some <emphasis>higher-order utility functions</emphasis> like fn:filter, fn:fold-left and fn:fold-right have changed as well. Please review your
-                use of those functions. Also, fn:map is now called fn:for-each, though the old name is still accepted.</para>
-            <para>The bundled Lucene has been upgraded from 3.6.1
-		to 4.4 with this release. Depending on what Lucene
-		analyzers you are using you need to change the
-		classnames in your
-		<filename>collection.xconf</filename>s. E.g. KeywordAnalyzer
-		and WhitespaceAnalyzer has moved into package
-		<filename>org.apache.lucene.analysis.core</filename>. Thus
-		change, any occurrence of
-		<filename>org.apache.lucene.analysis.WhitespaceAnalyzer</filename>
-		into
-		<filename>org.apache.lucene.analysis.core.WhitespaceAnalyzer</filename>
-		and all other moved classes in the collection
-		configurations and make sure you reindex your data
-		before use. You get an error notice in the
-		<filename>exist.log</filename> if you overlooked any
-		occurrences.</para>
-        </section>
-        <section>
-            <title>Upgrading to 2.1</title>
-            <para>The 2.1 release is not binary compatible with the 1.4.x series. You need to
-                backup/restore. 2.1 is binary compatible with 2.0 though.</para>
-        </section>
-        <section>
-            <title>Upgrading to 2.0</title>
-            <para>The 2.0 release is not binary compatible with the 1.4.x series. You need to
-                backup/restore.</para>
-            <section>
-                <title>Special Notes</title>
-                <variablelist>
-                    <varlistentry>
-                        <term>Permissions</term>
+        <section id="non-binary-compatible-upgrades">
+            <title>Non-binary-compatible Upgrades</title>
```

Comment:
the styling of this title is not in line with the other titles, how about ``Binary Non-compatible Upgrades``